packages/legacy/core/App/components/misc/NewQRView.tsx

```diff
@@ -174,17 +174,25 @@ const NewQRView: React.FC<Props> = ({ defaultToConnect, handleCodeScan, error, e
               {invitation && <QRRenderer value={invitation} size={qrSize} />}
             </View>
             <View style={{ paddingHorizontal: 20, flex: 1 }}>
-              <View style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'center' }}>
+              <View
+                style={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                }}
+              >
                 <Text testID={testIdWithKey('WalletName')} style={[styles.walletName, { paddingHorizontal: 20 }]}>
                   {store.preferences.walletName}
                 </Text>
-                <TouchableOpacity
+                <HeaderButton
```

Comment:
Should we rename this from HeaderButton since it's not only being used as a header button now?

Reply:
Yes, but I'd need to refactor some other parts of the app. I've created [#1807](https://github.com/bcgov/bc-wallet-mobile/issues/1807) to track this tech debt.